src/main/java/com/monitorjbl/xlsx/impl/StreamingSheetReader.java

```diff
@@ -39,6 +40,7 @@
   private final StylesTable stylesTable;
   private final XMLEventReader parser;
   private final DataFormatter dataFormatter = new DataFormatter();
+  private final Set<Integer> hiddenColumns = new HashSet<>();
```

Comment:
No, this does not apply to rows, this is in support of the org.apache.poi.ss.usermodel.Sheet.isColumnHidden(int columnIndex) method